scipy/stats/_stats_py.py

```diff
@@ -4642,28 +4642,82 @@ def fisher_exact(table, alternative='two-sided'):
+           Mathematics of a Lady Tasting Tea." ISBN 978-0-486-41151-4, 1935.
+    .. [2] "Fisher's exact test",
+           https://en.wikipedia.org/wiki/Fisher's_exact_test
+    .. [3] Emma V. Low et al. "Identifying the lowest effective dose of
+           acetazolamide for the prophylaxis of acute mountain sickness:
+           systematic review and meta-analysis."
+           BMJ, 345, :doi:`10.1136/bmj.e6779`, 2012.
+
     Examples
     --------
-    Say we spend a few days counting whales and sharks in the Atlantic and
-    Indian oceans. In the Atlantic ocean we find 8 whales and 1 shark, in the
-    Indian ocean 2 whales and 5 sharks. Then our contingency table is::
+    In [3]_, the effective dose of acetazolamide for the prophylaxis of acute
+    mountain sickness was investigated. The study notably concluded:
+
+        Acetazolamide 250 mg, 500 mg, and 750 mg daily were all efficacious for
+        preventing acute mountain sickness. Acetazolamide 250 mg was the lowest
+        effective dose with available evidence for this indication.
+
+    The following table summarizes the results of the experiment in which
+    some participants took a daily dose of acetazolamide 250 mg while others
+    took a placebo.
+    Cases of acute montain sickness were recorded::
+
+                                    Acetazolamide   Control/Placebo
+        Acute montain sickness             7           17
+        No                                15            5
+
+
+    Is there evidence that the acetazolamide 250 mg reduces the risk of
+    acute montain sickness?
+    We begin by formulating a null hypothesis :math:`H_0`:
 
-                Atlantic  Indian
-        whales     8        2
-        sharks     1        5
+        The effect of acetazolamide is equivalent to that of placebo and
+        does not reduce the risk of acute montain sickness.
 
-    We use this table to find the p-value:
+    Let's assess the plausibility of this hypothesis with
+    Fisher's test.
 
     >>> from scipy.stats import fisher_exact
-    >>> res = fisher_exact([[8, 2], [1, 5]])
+    >>> res = fisher_exact([[7, 17], [15, 5]], alternative='less')
+    >>> res.statistic
+    0.13725490196078433
     >>> res.pvalue
-    0.0349...
+    0.0028841933752349743
+
+    Using a significance level of 5%, we would reject the null hypothesis in
+    favor of the alternative hypothesis: "the effect of acetazolamide
+    is not equivalent to the effect of placebo and reduces the risk of
+    acute montain sickness."
+
+    .. note::
+
+        Because the null distribution of Fisher's exact test is formed under
+        the assumption that both row and column sums are fixed, the result of
+        the test are conservative when applied to an experiment in which the
+        row sums are not fixed.
+
+        In this case, the sum of columns is fixed. 22 persons are in both
+        groups. But the number of person getting sick in both groups is not
```

Comment:
To match the language in the preceeding paragraph (and correct the plurality "sum" - there are two columns, so two sums):
```suggestion
        In this case, the column sums are fixed; there are 22 subjects in each
        group. But the number of cases of acute mountain sickness is not
```